pkg/constants/constants.go

```diff
@@ -29,6 +29,7 @@ const (
 	LabelIsHeadless          = "lighthouse.submariner.io/is-headless"
 	PublishNotReadyAddresses = "lighthouse.submariner.io/publish-not-ready-addresses"
 	GlobalnetEnabled         = "lighthouse.submariner.io/globalnet-enabled"
+	KubernetesServiceName    = "kubernetes.io/service-name"
```

Comment:
There’s a constant for this in the Discovery API: https://pkg.go.dev/k8s.io/api/discovery/v1#pkg-constants (`LabelServiceName`).

Reply:
Ahh thanks.